packages/kit/src/core/dev/plugin.js

```diff
@@ -163,8 +163,13 @@ export async function create_plugin(config) {
 
 			update_manifest();
 
-			vite.watcher.on('add', update_manifest);
-			vite.watcher.on('unlink', update_manifest);
+			for (const event of ['add', 'unlink']) {
+				vite.watcher.on(event, (file) => {
+					if (file.startsWith(config.kit.files.routes + path.sep)) {
```

Comment:
honestly not sure. i'll push some logging to make it visible

Reply:
both `config.kit.files.routes` and the `file` use `\` — it's ok to use `path.sep`

Reply:
I just tried this on Windows, and to my surprise (and slight dismay), it uses backslashes in the paths emitted by events - even though the docs say the glob you pass to `chokidar.watch()` needs to use forward slashes.

In Vite it looks like forward slashes are being used in paths being sent _to_ chokidar, rather than in paths getting returned by chokidar.

Reply:
Whoops, thanks for not telling me about the previous comments, GitHub.